services/snyk/snyk-vulnerability-github.service.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const SynkVulnerabilityBase = require('./snyk-vulnerability-base')
+
+module.exports = class SnykVulnerabilityGitHub extends SynkVulnerabilityBase {
+  static get route() {
+    return {
+      base: 'snyk/vulnerabilities/github',
+      format: '([^/]+)/([^/]+)(?:/(.+))?',
+      capture: ['user', 'repo', 'manifestFilePath'],
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Snyk Vulnerabilities for GitHub Repo',
+        pattern: ':user/:repo',
+        namedParams: {
+          user: 'badges',
+          repo: 'shields',
+        },
+        staticExample: this.render({ vulnerabilities: '0' }),
+      },
+      {
+        title: 'Snyk Vulnerabilities for GitHub Repo (Specific Manifest)',
+        pattern: ':user/:repo/:manifestFilePath',
+        namedParams: {
+          user: 'badges',
+          repo: 'shields',
+          manifestFilePath: 'gh-badges/package.json',
+        },
+        staticExample: this.render({ vulnerabilities: '0' }),
+        documentation: `
+          <p>
+            Provide the direct path to your target manifest file. Snyk does not
+            support using a specific branch for this, so do not include "blob" nor a branch name.
+          </p>
+          `,
+      },
+    ]
+  }
+
+  async handle({ user, repo, manifestFilePath }) {
+    const url = `https://snyk.io/test/github/${user}/${repo}/badge.svg`
+    const qs = {}
+
+    if (manifestFilePath) {
+      qs.targetFile = manifestFilePath
```

Comment:
This is fine, though for future reference you can write this inline without the conditional:

```js
  qs: { targetFile: manifestFilepath },
```

Reply:
Good to know (I've been using those conditionals in a few places). I take it that means that `request` is smart enough to not query string params if the values are falsy?